neurodsp/aperiodic/conversions.py

```diff
@@ -0,0 +1,255 @@
+    exponent = -exponent
+
+    if fractional_class == 'gaussian':
+        hurst = (exponent + 1) / 2.
+    elif fractional_class == 'brownian':
+        hurst = (exponent - 1) / 2.
+
+    return hurst
+
+def convert_hurst_exponent(hurst, fractional_class):
+    """Convert a Hurst exponent value to the expected powerlaw exponent.
+
+    Parameters
+    ----------
+    hurst : float
+        Hurst exponent value.
+    fractional_class : {'gaussian', 'brownian'}
+        The class of input data that the given exponent value relates to.
+        This can be either 'fractional Gaussian noise' or 'fractional Brownian motion.'
+        This is required as the conversion differs between the two classes.
+
+    Returns
+    -------
+    exponent : float
+        Predicted aperiodic exponent value, representing a 1/f distribution.
+
+    References
+    ----------
+    .. [1] Schaefer, A., Brach, J. S., Perera, S., & Sejdić, E. (2014). A comparative analysis
+           of spectral exponent estimation techniques for 1/fβ processes with applications to
+           the analysis of stride interval time series. Journal of Neuroscience Methods, 222, 118–130.
+           https://doi.org/10.1016/j.jneumeth.2013.10.017
+
+    Examples
+    --------
+    Convert a Hurst exponent value to the expected powerlaw exponent, for fractional Gaussian noise:
+
+    >>> convert_hurst_exponent(1, 'gaussian')
+    -1.0
+
+    Convert a Hurst exponent value to the expected powerlaw exponent, for fractional Brownian noise:
+
+    >>> convert_hurst_exponent(0, 'brownian')
+    -1.0
+    """
+
+    check_param_options(fractional_class, 'fractional_class', ['gaussian', 'brownian'])
+
+    if fractional_class == 'gaussian':
+        exponent = 2. * hurst - 1
+    elif fractional_class == 'brownian':
+        exponent = 2. * hurst + 1
+
+    # Convert predicted exponent value to have a negative sign
+    exponent = -exponent
+
+    return exponent
+
+
+def convert_exponent_hfd(exponent):
+    """Convert exponent to expected Higuchi fractal dimension value.
+
+    Parameters
+    ----------
+    exponent : float
+        Aperiodic exponent value, representing a 1/f distribution.
+
+    Returns
+    -------
+    hfd : float
+        Predicted Higuchi fractal dimension value.
+
+    Notes
+    -----
+    This works for exponents between [-1, -3] (inclusive).
+    As a special case, if exponent is 0, Higuchi Fractal Dimension is 2.
+
+    References
+    ----------
+    .. [1] Cervantes-De la Torre, F., González-Trejo, J. I., Real-Ramírez, C. A., &
+           Hoyos-Reyes, L. F. (2013). Fractal dimension algorithms and their application
+           to time series associated with natural phenomena. Journal of Physics: Conference
+           Series, 475, 012002. https://doi.org/10.1088/1742-6596/475/1/012002
+
+    Examples
+    --------
+    Convert exponent value to the expected Higuchi fractal dimension:
+
+    >>> convert_exponent_hfd(-1)
+    2.0
```

Comment:
Under the notes header, it says if exponent = 0, hfd = 2. And then in this example when exponent = -1, hfd = 2.

Reply:
Oops, maybe both are correct. So 0 hfd == -1 hfd == 0 exponent?